File: avalon-support/avalon-oauth2-support/avalon-oauth2-authorization-server/src/test/java/io/github/lishangbu/avalon/oauth2/authorizationserver/exception/InvalidCaptchaExceptionTest.java
package io.github.lishangbu.avalon.oauth2.authorizationserver.exception;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

class InvalidCaptchaExceptionTest {

    @Test
    void exposesMessage() {
        InvalidCaptchaException exception = new InvalidCaptchaException("captcha");

        assertEquals("captcha", exception.getMessage());
    }
}

File: avalon-platform/avalon-security/avalon-oauth2-authorization-server/src/test/kotlin/io/github/lishangbu/avalon/oauth2/authorizationserver/exception/InvalidCaptchaExceptionTest.kt
package io.github.lishangbu.avalon.oauth2.authorizationserver.exception

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class InvalidCaptchaExceptionTest {
    @Test
    fun exposesMessage() {
        val exception = InvalidCaptchaException("captcha")

        assertEquals("captcha", exception.message)
    }
}
